tidy3d/material_library/material_library.py

```diff
@@ -80,6 +82,80 @@ def medium(self):
         return self.variants[self.default].medium
 
 
+class Variant2D(Tidy3dBaseModel):
+    """Reference, data_source, and material model for a variant of a 2D material."""
+
+    medium: Medium2D = pd.Field(
+        ...,
+        title="Material dispersion model",
+        description="A dispersive medium described by the pole-residue pair model.",
+    )
+
+    medium_parametric: Callable = pd.Field(
+        None,
+        title="Parametric material model",
+        description="Parameteric 2d material dispersion model.",
+    )
+
+    reference: List[ReferenceData] = pd.Field(
+        None,
+        title="Reference information",
+        description="A list of reference related to this variant model.",
```

Comment:
references (plural) and can you fix if this is a typo in the original material library too?